src/SassBinary.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Symfonycasts\SassBundle;
+
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Component\Process\Process;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class SassBinary
+{
+    private const VERSION = '1.63.6';
```

Comment:
how about if this was configurable by the user?
that way, the user can upgrade sass, without waiting for a new release of the bundle.

Reply:
Seems reasonable to me - the constant would be the default. It wouldn't surprise me also if we got a request in the future where someone needed to "lock" to an older version.